pypgstac/setup.py

```diff
@@ -6,13 +6,13 @@
     desc = f.read()
 
 install_requires = [
-    "smart-open==4.2.*",
+    "smart-open==6.2.*",
```

Comment:
Hard requirements can be dangerous, IMO we should try to have versions opened to a specific major release (e.g `>=6.0,<7.0`)

Reply:
but if it worked with 4.2 we could do `>=4.2,<7.0`

Reply:
Good idea! I will update the strict requirements to be more permissive.